service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -209,7 +220,8 @@ void testCreateDatasetWithInvalidUser() {
 
   @Test
   void testCreateDataset() {
-    String metadata = "test metadata";
+    String metadata = """
```

Comment:
May as well make a string constant for the duplicate metadata used in these tests.